src/ucp/core/ucp_ep.c

```diff
@@ -2812,6 +2812,18 @@ int ucp_ep_do_keepalive(ucp_ep_h ep)
     return 0;
 }
 
+static void ucp_ep_req_purge_send(ucp_request_t *req, ucs_status_t status)
+{
+    if ((status == UCS_OK) ||
+        (ucp_ep_config(req->send.ep)->key.err_mode ==
+         UCP_ERR_HANDLING_MODE_NONE) ||
+        !UCP_DT_IS_CONTIG(req->send.datatype)) {
```

Comment:
add flag that we sent remote ket to peer
UCP_REQUEST_FLAG_RKEY_INUSE

Reply:
added flag